tests/php/TemplatingTest.php

```diff
@@ -330,10 +241,8 @@ public function templateWithFilterAccessInAttributeBinding_CorrectValueIsRendere
 		assertThat( $result, is( equalTo( '<p attr1="VALUE"></p>' ) ) );
 	}
 
-	/**
-	 * @test
-	 */
-	public function templateWithFilterAccessInAttributeBindingInsideTheLoop_CorrectValueIsRendered() {
+	// phpcs:ignore Generic.Files.LineLength.TooLong
```

Comment:
I suggest to increase the line length in .phpcs.xml instead. Wikibase for example uses 140 for years. This doesn't mean lines _should_ be that long. Devs are still encouraged to stop at 100. But PHPCS will not complain if there is an occasional exception, like here.